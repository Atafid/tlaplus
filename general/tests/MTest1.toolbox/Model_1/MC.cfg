\* CONSTANT declarations
a = a
SPECIFICATION spec_12495080839932000
\* Generated at Wed Aug 05 14:34:43 PDT 2009