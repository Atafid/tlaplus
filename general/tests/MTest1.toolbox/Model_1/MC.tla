---- MODULE MC ----
EXTENDS MTest1, TLC

\* Specification formula
spec_12495080839932000 ==
x = /\[][ x' = x+ ]_<<x>> 
----
====
\* Generated at Wed Aug 05 14:34:43 PDT 2009