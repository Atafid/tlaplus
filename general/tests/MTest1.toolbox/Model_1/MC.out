@!@!@STARTMSG 1000:1 @!@!@
@!@!@STARTMSG 5006:0 @!@!@
TLC found an error in the configuration file at line 2
It was expecting a keyword, but did not find it.
@!@!@ENDMSG 5006 @!@!@
@!@!@ENDMSG 1000 @!@!@
